projects/batfish/src/main/java/org/batfish/representation/iptables/IptablesChain.java

```diff
@@ -19,7 +19,7 @@
 
   private String _name;
 
-  private ChainPolicy _policy;
```

Comment:
For consistency, can you please move this to the constructor, where `_name` and `_rules` are initialized?